src/dynamic_program.rs

```diff
@@ -1,32 +1,25 @@
-extern crate elf;
-extern crate rbpf;
+// TODO rename to native_loader
```

Comment:
bah, let's not add this TODO.  File an issue or remember :)

Reply:
Yeah, its just there for this preview